src/shogun/features/StringFeatures.cpp

```diff
@@ -1731,6 +1734,8 @@ template<class ST> void CStringFeatures<ST>::init()
 
 	m_parameters->add_vector(&symbol_mask_table, &symbol_mask_table_len, "mask_table", "Symbol mask table - using in higher order mapping");
 	watch_param("mask_table", &symbol_mask_table, &symbol_mask_table_len);
+	watch_method("get_num_vectors", &CStringFeatures::get_num_vectors);
```

Comment:
I will have to try again the "features" because I was getting a double registration error. But I will let you know.

Reply:
Ah yes. It is actually awkward as "features" is the registered raw array of `SGString` ... sigh.
"get_features" is actually a bad name as it simply converts the array to StringList. "string_list" might be the best name.

Eventually, it would be best to somehow refactor/unify all this. But not for now. Now is just for adding the string list to the API

Reply:
I can change it to string_list for now but this will affect a few files.